mava/systems/tf/maddpg/system.py

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""MADDPG system implementation."""
+"""MADDPG scaled system implementation."""
```

Comment:
Remove "scaled"? Since this will become the default.

Reply:
Fixed it.